example/StackNavigator.tsx

```diff
@@ -53,7 +53,6 @@ export type StackNavigationProp<
    */
   popToTop(): void;
 };
-
 const StackRouter: Router<CommonAction | Action> = {
```

Comment:
The router should also merge base router no? Then we can remove the methods those are the same